<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNewcurve">
     <property name="text">
      <string>Create new curve</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
